velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -266,6 +269,25 @@ TEST_F(StringTest, left) {
   EXPECT_EQ(left("da\u6570\u636Eta", 30), "da\u6570\u636Eta");
 }
 
+TEST_F(StringTest, right) {
```

Comment:
Please arrange the tests following alphabetical order.

Reply:
Updated